configure.ac

```diff
@@ -632,7 +632,7 @@ echo
 echo '* Checking for apt_pkg library used by: dpkginfo '
 PKG_CHECK_MODULES([apt_pkg], [libapt-pkg >= 0.0],[],[
 SAVE_LIBS=$LIBS
-AC_SEARCH_LIBS([pkgInitConfig],[apt-pkg],[
+AC_SEARCH_LIBS([pkgVersion],[apt-pkg],[
```

Comment:
You need to edit ac_probes/configure.ac.tpl as well. And make sure that they are in sync by running `./confgen.sh`